### PR TITLE
Create new payment queue in bulk scan namespace BPS-764

### PR DESCRIPTION
**Pull request by mdayican**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/BPS-764

### Change description ###

Create new payment queue in bulk scan namespace 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by mdayican** [comment]
> key_vault_id


> looks ok. can we start using `key_vault_id` instead of `vault_uri`?


replaced

File: queue.tf [hunk 1]
```diff
@@ -36,48 +36,71 @@ module "processed-envelopes-queue" {
...
+}
+
 # region connection strings and other shared queue information as Key Vault secrets
 resource "azurerm_key_vault_secret" "envelopes_queue_send_conn_str" {
-  name      = "envelopes-queue-send-connection-string"
-  value     = "${module.envelopes-queue.primary_send_connection_string}"
-  vault_uri = "${data.azurerm_key_vault.key_vault.vault_uri}"
+  name          = "envelopes-queue-send-connection-string"
+  value         = "${module.envelopes-queue.primary_send_connection_string}"
+  key_vault_id  = "${data.azurerm_key_vault.key_vault.vault_uri}"
```
Comment (nitinprabhu) [inline queue.tf]: all these key vault changes should be ideally a separate PR ?

Comment (doncem) [reply queue.tf]: ideally, but it's ok here atm
